azurelinuxagent/common/osutil/freebsd.py

```diff
@@ -93,6 +97,306 @@ def get_if_mac(self, ifname):
     def get_first_if(self):
         return self._get_net_info()[:2]
 
+    @staticmethod
+    def read_route_table():
+        """
+        Return a list of strings comprising the route table as in the Linux /proc/net/route format. The input taken is from FreeBSDs
+        `netstat -rn -f inet` command. Here is what the function does in detail:
+
+        1. Runs `netstat -rn -f inet` which outputs a column formatted list of ipv4 routes in priority order like so:
+
+            > Routing tables
+            > 
+            > Internet:
+            > Destination        Gateway            Flags    Refs      Use  Netif Expire
+            > default            61.221.xx.yy       UGS         0      247    em1
+            > 10                 10.10.110.5        UGS         0       50    em0
+            > 10.10.110/26       link#1             UC          0        0    em0
+            > 10.10.110.5        00:1b:0d:e6:58:40  UHLW        2        0    em0   1145
+            > 61.221.xx.yy/29    link#2             UC          0        0    em1
+            > 61.221.xx.yy       00:1b:0d:e6:57:c0  UHLW        2        0    em1   1055
+            > 61.221.xx/24       link#2             UC          0        0    em1
+            > 127.0.0.1          127.0.0.1          UH          0        0    lo0
+        
+        2. Convert it to an array of lines that resemble an equivalent /proc/net/route content on a Linux system like so:
+
+            > Iface   Destination Gateway     Flags   RefCnt  Use Metric  Mask        MTU Window  IRTT
+            > gre828  00000000    00000000    0001    0   0   0   000000F8    0   0   0
+            > ens160  00000000    FE04700A    0003    0   0   100 00000000    0   0   0
+            > gre828  00000008    00000000    0001    0   0   0   000000FE    0   0   0
+            > ens160  0004700A    00000000    0001    0   0   100 00FFFFFF    0   0   0
+            > gre828  2504700A    00000000    0005    0   0   0   FFFFFFFF    0   0   0
+            > gre828  3704700A    00000000    0005    0   0   0   FFFFFFFF    0   0   0
+            > gre828  4104700A    00000000    0005    0   0   0   FFFFFFFF    0   0   0
+
+        :return: Entries in the ipv4 route priority list from `netstat -rn -f inet` in the linux `/proc/net/route` style
+        :rtype: list(str)
+        """
+            
+        def _get_netstat_rn_ipv4_routes():
+            """
+            Runs `netstat -rn -f inet` and parses its output and returns a list of routes where the key is the column name
+            and the value is the value in the column, stripped of leading and trailing whitespace.
+
+            :return: List of dictionaries representing routes in the ipv4 route priority list from `netstat -rn -f inet`
+            :rtype: list(dict)
+            """
+            cmd = "netstat -rn -f inet"
+            ret, output = shellutil.run_get_output(cmd)
```

Comment:
we are deprecating shellutil.run_get_output in favor of shellutil.run_command, could you change this?

Reply:
Done with a0112e2 and 72bddb7